scripts/register_files.py

```diff
@@ -0,0 +1,259 @@
+"""
+Simple generic script to register observation files to r2d2 v3
+Works for ncdiag, odas, gdas_marine, etc.
+"""
+
+import os
+import sys
+import glob
+
+# Color codes for printing
+RED = "\033[91m"
+GREEN = "\033[92m"
+YELLOW = "\033[93m"
+BLUE = "\033[94m"
+RESET = "\033[0m"
+
+try:
+    import r2d2
+except ImportError as e:
```

Comment:
I like the helpful message. You can also handle this by raising an error with a custom message, which provides us with a more consistent way of handling issues
```suggestion
try:
    import r2d2
except ImportError:
    raise ImportError(f'Failed to import r2d2, use \nLoad module: module load r2d2-client/sles15_0604')
```